samples/apps/wayfinder-sample/README.md

```
@@ -157,7 +157,8 @@ The agent's client secret defaults to `wayfinder-agent-secret` (set in `thunderi
 | `JOHN_DOE_PASSWORD` | Yes | Password for the `john.doe` demo user |
 | `JANE_SMITH_PASSWORD` | Yes | Password for the `jane.smith` demo user |
 | `ALEX_CARTER_PASSWORD` | Yes | Password for the `alex.carter` demo user |
-| `CIBA_SMS_SENDER_ID` | **SMS only** | Notification sender ID for the CIBA SMS flow. Required only if you switch the upgrade agent to `wayfinder-ciba-sms-flow`. Obtain the sender ID after running the SMS setup script (see [CIBA Upgrade — SMS channel](#ciba-upgrade--sms-channel) below). |
+| `CIBA_LOGIN_HINT_ATTRIBUTE` | Yes | Attribute used by the CIBA flow's `IdentifyingExecutor` to resolve the login hint to a user. Set to `email` (default) when using `login_hint`, or `userID` when using `id_token_hint`. |
+| `SMS_SENDER_ID` | **SMS only** | Notification sender ID for the CIBA SMS flow. Required only if you switch the upgrade agent to `wayfinder-ciba-sms-flow`. Obtain the sender ID after running the SMS setup script (see [CIBA Upgrade — SMS channel](#ciba-upgrade--sms-channel) below). |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)